src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/ResourceGroup.kt

```diff
@@ -0,0 +1,7 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
```

Comment:
Since our conversation this evening, it seems like ResourceGroup should be in ViewModel instead of Model. Thoughts?

Reply:
It's a utility class used by the UI I presume. It would belong in the domain.